test/index.spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint no-unused-expressions: 0 */
+/* eslint no-console: 0 */
+/* eslint no-global-assign: 0 */
+
+import { makeConsoleMock } from 'consolemock'
+import lintStaged from '../src/index'
+
+jest.mock('cosmiconfig', () => jest.fn(() => Promise.resolve({ config: { verbose: true } })))
```

Comment:
Implementation for the mock is not required here as that is provided in each test. You can replace this with `jest.mock('cosmiconfig')`. I tested this, works fine.